server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -98,6 +99,22 @@ const accountMutations = {
           }
         }
 
+        await models.Activity.create(
+          {
+            type: activities.COLLECTIVE_EDITED,
+            UserId: req.remoteUser.id,
+            UserTokenId: req.userToken?.id,
+            CollectiveId: account.id,
+            FromCollectiveId: account.id,
+            HostCollectiveId: account.approvedAt ? account.HostCollectiveId : null,
+            data: {
+              previousData: { settings: { [args.key]: account.data?.[args.key] } },
+              newData: { settings: { [args.key]: args.value } },
+            },
+          },
+          { transaction },
+        );
```

Comment:
`account.update` can fail because of the model validations. The activity should be created only if it succeeds.

Reply:
Thanks. I've corrected that as well. 👍🏽